iot-e2e-tests/common/src/main/java/com/microsoft/azure/sdk/iot/common/tests/iothubservices/TransportClientTests.java

```diff
@@ -644,7 +644,16 @@ public void invokeMethodAMQPSWSInvokeParallelSucceed() throws Exception
     @Test
     public void testTwin() throws IOException, InterruptedException, IotHubException, URISyntaxException
     {
-        TransportClient transportClient = setUpTwin();
+        TransportClient transportClient = null;
```

Comment:
I would suggest to move this into `setUpTwin` function instead of swallow it with `fail`.
```
void setUpTwin() throws ...Exception {
   try {
   .....
   } catch(...Exception e) {
      log.error("Encountered exception during setUpTwin.");
      throw e;
   }
}
```